src/mscorlib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -1060,7 +1060,7 @@ public static IntPtr GetHINSTANCE(Module m)
             }
 
             if (rtModule == null)
-                throw new ArgumentNullException(Environment.GetResourceString("Argument_MustBeRuntimeModule"));
+                throw new ArgumentNullException("m", Environment.GetResourceString("Argument_MustBeRuntimeModule"));
```

Comment:
Unrelated to your change, but this doesn't seem like it should be a ArgumentNullException.